src/applications/facility-locator/constants/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
Can we add this comment just surrounding the block with camel case properties?

Reply:
Moved 👍